RecoHI/HiTracking/python/MergeTrackCollectionsHI_cff.py

```diff
@@ -86,6 +86,9 @@
                       cms.InputTag('hiDetachedTripletStepTracks'),
                       cms.InputTag('hiLowPtTripletStepTracks'),
                       cms.InputTag('hiPixelPairGlobalPrimTracks'),
+                      cms.InputTag('hiMixedTripletStepTracks'),
+		      cms.InputTag('hiPixelLessStepTracks'),
+		      cms.InputTag('hiTobTecStepTracks'),
```

Comment:
the indentation is inconsistent

Reply:
Seems like a bug on git to diff the code. The indentation is consistent if you click into the file, https://github.com/cmsHiTracking/cmssw/blob/bb4fa82ee7144187e8b8da40039ab680ac64607c/RecoHI/HiTracking/python/MergeTrackCollectionsHI_cff.py

Reply:
Line 89 is intended with spaces, while lines 90 and 91 have a mixture of spaces and tabs.